src/tests/multihost/ad/test_adparameters.py

```diff
@@ -702,28 +703,47 @@ def test_0018_bz1734040(self, multihost, adjoin):
         """
         adjoin(membersw='adcli')
         client = sssdTools(multihost.client[0])
-        domain_name = client.get_domain_section_name()
         client.backup_sssd_conf()
         client.remove_sss_cache('/var/log/sssd')
-        sssdcfg = multihost.client[0].get_file_contents(SSSD_DEFAULT_CONF)
-        sssdcfg = re.sub(b'ad_domain = %s' % domain_name.encode('utf-8'),
-                         b'ad_domain = example.com \ndebug_level = 9', sssdcfg)
-        multihost.client[0].put_file_contents(SSSD_DEFAULT_CONF, sssdcfg)
+        domain_name = client.get_domain_section_name()
+        dom_section = 'domain/%s' % domain_name
+        client.sssd_conf(dom_section, {'debug_level': '9'})
+        client.sssd_conf('sssd', {'debug_level': '9'})
+
+        # Configure local files domain
+        local_params = {'id_provider': 'files', 'debug_level': '9'}
+        client.sssd_conf('domain/local', local_params)
+
+        # Make SSSD AD offline
+        multihost.client[0].run_command(
+            'which iptables || yum install -y iptables',
+            raiseonerr=False
+        )
+        multihost.client[0].run_command(
+            f'iptables -F; iptables -A INPUT -s {multihost.ad[0].ip} -j DROP;'
+            f'iptables -A OUTPUT -d {multihost.ad[0].ip} -j DROP',
+            raiseonerr=False
+        )
+
         client.clear_sssd_cache()
-        cmd = multihost.client[0].run_command('getent passwd 0',
-                                              raiseonerr=True)
-        if cmd.returncode != 0:
-            status = 'FAIL'
-        else:
-            status = 'PASS'
-        time.sleep(10)
+        (aduser, _) = create_aduser_group
+        cmd = multihost.client[0].run_command(
```

Comment:
SSSD doesn't handle root / 0, we don't need this.

Reply:
done